src/components/AddNewComment.jsx

```diff
@@ -0,0 +1,51 @@
+import { useState } from "react";
+import * as Api from "../utils/newsApi";
+
+export const AddNewComment = ({ article_id, comments, setComments }) => {
+  const [addNewComment, setAddNewComment] = useState({
+    username: "jessjelly",
+    votes: 0,
+    body: "",
+    created_at: "just now",
+  });
+  const [textValue, setTextValue] = useState("");
+
+  const handleCommentSubmit = (e) => {
+    e.preventDefault();
+    setAddNewComment({
+      username: "jessjelly",
+      body: textValue,
+      votes: 0,
+      created_at: "just now",
+    });
+    if (addNewComment.body === "") {
+      return;
+    }
+
+    setComments([addNewComment, ...comments]);
+    Api.postComment(article_id, addNewComment).then(() => {
+     
+      setTextValue("");
+    });
+  };
+
+  return (
+    <form onSubmit={handleCommentSubmit}>
+      <p>Comment:</p>
+      <div className="AddComment_container">
+        <div className="Username_input">
+          <label>UserName</label>
+          <input name="username" defaultValue={addNewComment.username}></input>
```

Comment:
This input needs to be a controlled component. At the moment, the browser is in charge of it and react doesnt know what inside it at all times.